pkg/events/controller/controller.go

```diff
@@ -0,0 +1,49 @@
+package controller
+
```

Comment:
I don't think this needs to be in its own package. If we move it up a directory then it would be referenced as `event.Controller` rather than `controller.Controller`.